### PR TITLE
Docs: mention `docsify` on "Build customization"

### PR DESCRIPTION
**Pull request by humitos**
Initial minimal mention to `docsify` as an example in our "Build customization"
section.

Related #9435

### CODE REVIEW

File: docs/user/build-customization.rst [hunk 1]
```diff
@@ -256,8 +256,23 @@ If your project requires full control of the build process,
 and :ref:`extending the build process <build-customization:extend the build process>` is not enough,
 all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands` configuration file key.
 
-For example, if your project uses `Pelican <https://blog.getpelican.com/>`_ instead of Sphinx for its documentation,
-your project could use the following configuration file:
+As Read the Docs does not have control over the build process,
+you are responsible for running all the commands required to install requirements and build your project properly.
+Once the build process finishes, the content of the ``_readthedocs/html/`` directory will be hosted.
```
Comment (agjohnson) [inline docs/user/build-customization.rst:261]:
```suggestion
Once the build process finishes, the contents of the ``_readthedocs/html/`` directory will be hosted.
```

File: docs/user/build-customization.rst [hunk 2]
```diff
@@ -256,8 +256,23 @@ If your project requires full control of the build process,
...
+
+
+Examples
+++++++++
+
+This section contains some examples that can be seen as showcase of what it's possible to do with :ref:`config-file/v2:build.commands`.
```
Comment (agjohnson) [inline docs/user/build-customization.rst:267]: Some reduction for clarity

```suggestion
This section contains some examples that showcase what is possible with :ref:`config-file/v2:build.commands`.
```

File: docs/user/build-customization.rst [hunk 3]
```diff
@@ -256,8 +256,23 @@ If your project requires full control of the build process,
...
+Note that you may need to modify and adapt them depending your needs.
```
Comment (agjohnson) [inline docs/user/build-customization.rst:268]:
```suggestion
Note that you may need to modify and adapt these examples depending on your needs.
```

File: docs/user/build-customization.rst [hunk 4]
```diff
@@ -272,9 +287,24 @@ your project could use the following configuration file:
        - pelican --settings docs/pelicanconf.py --output _readthedocs/html/ docs/
 
 
-As Read the Docs does not have control over the build process,
-you are responsible for running all the commands required to install requirements and build the documentation properly.
-Once the build process finishes, the ``_readthedocs/html/`` folder will be hosted.
+Docsify
+~~~~~~~
+
+`Docsify <https://docsify.js.org/>`_ generates documentation websites on the fly, without the need of building static HTML.
```
Comment (agjohnson) [inline docs/user/build-customization.rst:293]:
```suggestion
`Docsify <https://docsify.js.org/>`_ generates documentation websites on the fly, without the need to build static HTML.
```

**Review by agjohnson — APPROVED** [review]
Looks great! Just some copy notes.

At some point, we may want to think about restructuring this information to make finding these micro guides easier -- multiple pages or something. I'm sure we'll continue to accumulate a lot of examples here. Just worth thinking about for now though.

File: docs/user/build-customization.rst (hunk 2 above)
Comment (humitos) [reply docs/user/build-customization.rst:267]: I force myself to reduce verbosity while writing, but it seems that being Argentinian always wins here 😂

**Comment by shayded-exe** [comment]
Thanks for this! Docsify is currently the best docs tool out there in my opinion.

**Pull request merged**